web/src/lib/components/shared-components/settings/setting-switch.svelte

```diff
@@ -4,20 +4,23 @@
   import { createEventDispatcher } from 'svelte';
   import Slider from '$lib/components/elements/slider.svelte';
 
+  export let id: string;
   export let title: string;
   export let subtitle = '';
   export let checked = false;
   export let disabled = false;
   export let isEdited = false;
 
+  const sliderId = `${id}-slider`;
```

Comment:
Reactivity breaks when the `id` prop is updated

```suggestion
  $: sliderId = `${id}-slider`;
```